warehouse/__about__.py

```diff
@@ -37,7 +37,7 @@
     __commit__ = None
 
 __author__ = "The Python Packaging Authority"
-__email__ = "donald@stufft.io"
+__email__ = "ernest@python.org"
```

Comment:
Hmm, maybe we should make this `admin@mail.pypi.org` instead?

Reply:
Fixed. Ready for re-review.